libs/blocks/quiz-entry/utils.js

```diff
@@ -1,5 +1,8 @@
 import { getNormalizedMetadata } from '../quiz/utils.js';
 
+let analyticsType;
```

Comment:
These values should be handled like all the rest of these, a const in the getQuizEntryData function that is returned. Local variables in utils is the primary reason why I couldn't use more from the quiz block. 

I have given preference to leaving functions that are using app properties (like getAnalyticsDataForBtn) inside of the app, rather than move them to utils where you would need to use local variables like this. Using "export const getAnalyticsDataForBtn" won't actually be useful in other blocks outside of quiz-entry, just the same as the issues that I ran into in trying to use more from quiz/utils.